README.md

```diff
@@ -1,22 +1,17 @@
-# PerconaMigrator [![Build Status](https://travis-ci.org/redbooth/percona_migrator.svg?branch=master)](https://travis-ci.org/redbooth/percona_migrator) [![Code Climate](https://codeclimate.com/github/redbooth/percona_migrator/badges/gpa.svg)](https://codeclimate.com/github/redbooth/percona_migrator)
+# Percona Migrator [![Build Status](https://travis-ci.org/redbooth/percona_migrator.svg?branch=master)](https://travis-ci.org/redbooth/percona_migrator) [![Code Climate](https://codeclimate.com/github/redbooth/percona_migrator/badges/gpa.svg)](https://codeclimate.com/github/redbooth/percona_migrator)
 
-Percona Migrator is a tool for running online and non-blocking
-DDL `ActiveRecord::Migrations` using `pt-online-schema-change` command-line tool of
+Percona Migrator is an **ActiveRecord connection adapter** that allows running
+**MySQL online and non-blocking DDL** `ActiveRecord::Migration`s without needing
+    to use a different DSL other than Rails' migrations DSL.
+
+It uses `pt-online-schema-change` command-line tool of
 [Percona
 Toolkit](https://www.percona.com/doc/percona-toolkit/2.0/pt-online-schema-change.html)
-which supports foreign key constraints.
-
-It adds a `db:percona_migrate:up` runs your migration using the
-`pt-online-schema-change` command.  It will apply exactly the same changes as
-if you run it with `db:migrate:up` avoiding deadlocks and without the need to
-change how you write regular rails migrations.
-
-It also disables `rake db:migrate:up` for the ddl migrations on envs with
-PERCONA_TOOLKIT var set to ensure all these migrations use Percona in production.
+which runs MySQL alter table statements without downtime.
 
 ## Installation
 
-Percona Migrator relies on `pt-online-schema-change` from  [Percona
+Percona Migrator relies on `pt-online-schema-change` from [Percona
 Toolkit](https://www.percona.com/doc/percona-toolkit/2.0/pt-online-schema-change.html)
 
 For mac, you can install it with homebrew typing `brew install percona-toolkit`. For
```

Comment:
Ufff, can you please guys add the installation steps for your platforms? I don't have an easy way to reproduce the steps :trollface: